package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I'll leave this in for now, since I'm going to use it to create a helper script for adding new translations